tools/ratbagctl.in

```diff
@@ -719,7 +719,7 @@ parser_def = [
                     func: func_led_get,
                 },
                 {
-                    of_type: set,
+                    of_type: switch,
```

Comment:
ok, found it in my history:
`sudo ./build/ratbagctl.devel led 0 set color ffffff` triggers `AttributeError: 'Namespace' object has no attribute 'func'`. the event node is missing, so it shouldn't succeed but that error is not ideal. also happens for a bunch of invalid commands (e.g. just using `led` or `led 0`). 

fwiw, `set color` is supported by ratbag-command. Anyway, I'll drop this commit for now